fvm/state/accounts.go

```diff
@@ -282,11 +274,110 @@ func (a *Accounts) SetCode(address flow.Address, code []byte) error {
 		return nil
 	}
 
-	a.ledger.Set(string(address.Bytes()), string(address.Bytes()), keyCode, code)
+	return a.setValue(address, true, keyCode, code)
+}
+
+// GetStorageUsed returns the amount of storage used in bytes by this account
+func (a *Accounts) GetStorageUsed(address flow.Address) (uint64, error) {
+	storageUsedRegister, err := a.getValue(address, false, keyStorageUsed)
+	if err != nil {
+		return 0, err
+	}
+
+	if len(storageUsedRegister) != uint64StorageSize {
+		return 0, fmt.Errorf("account %s storage used is not initialized or not initialized correctly", address.Hex())
+	}
+
+	storageUsed := binary.LittleEndian.Uint64(storageUsedRegister)
```

Comment:
We're normally storing data in big-endian format, so should maybe stay consistent here. 
There seem to be utilities in https://github.com/onflow/flow-go/blob/e7d87c2a8e2fc3e1ef6613729c7ee7ff51467810/ledger/common/utils/utils.go

Reply:
thank you! Missed those.